dom.go

```diff
@@ -149,8 +149,15 @@ type HTML struct {
 	lastRenderedChild *HTML
 }
 
-// Node returns the underlying JavaScript Element or TextNode.
-func (h *HTML) Node() *js.Object { return h.node.(wrappedObject).j }
+// Node returns the underlying JavaScript Element or TextNode. Node panics
+// when called before the associated component's Mount phase.
+func (h *HTML) Node() *js.Object {
+	if h.node == nil {
+		panic("(*HTML)Node() called before node creation. " +
+			"The component should be mounted before using Node().")
```

Comment:
A few things:

1. All Vecty panics should start with `vecty: `, akin to how some Go stdlib packages do it.
2. The string concatenation here is needless.
3. `(*HTML)Node()` is more correctly written as `(*HTML).Node()` I think.
4. It's weird to reference it as `(*HTML)Node()` at the start of the panic message, but as just `Node()` later on in the panic message.

I suggest writing this as just:

```Go
panic("vecty: cannot call (*HTML).Node() before node creation / component mount")
```